Sources/BothamPullToRefreshHandler.swift

```diff
@@ -36,8 +39,14 @@ public class BothamPullToRefreshHandler: NSObject {
         self.refreshControl.endRefreshing()
     }
 
-    public func beginRefreshing(scrollView: UIScrollView) {
-        scrollView.setContentOffset(CGPoint(x: 0, y: -refreshControl.frame.size.height), animated: true)
-        self.refreshControl.beginRefreshing()
+    public func beginRefreshing() {
+        if !refreshControl.refreshing {
+            scrollView?.setContentOffset(CGPoint(x: 0, y: -refreshControl.frame.size.height), animated: true)
+            self.refreshControl.beginRefreshing()
+        }
+    }
+    
+    public func isRefreshing() -> Bool {
+        return self.refreshControl.refreshing
```

Comment:
@davideme if we accept these changes we will have to generate a new major version? Do we have any pending branch to be published in a major? I think this implementation is ok but we have to keep in mind the breaking change.